src/components/index-page/IndexBuiltForYou.tsx

```diff
@@ -1,38 +1,57 @@
 import React from "react";
 
-import decorativeSquare from "../../assets/images/decorative_square.png";
+import { StaticImage } from "gatsby-plugin-image";
+
 import "./css/built-for-you.css";
 
 export const IndexBuiltForYou = () => (
   <section className="built-for-you">
     <div className="built-for-you__contents">
-      <img
+      <StaticImage
         alt=""
         className="built-for-you__decorative-square"
         role="presentation"
-        src={decorativeSquare}
+        src="../../images/decorations/decorative_square.png"
       />
       <div className="built-for-you__heading-container">
         <h2 className="built-for-you__heading">A partnership built for you</h2>
       </div>
       <div className="built-for-you__cards-container">
         <div className="built-for-you__card">
           <div className="built-for-you__image-container">
-            <img className="built-for-you__image" src="https://placehold.co/336x160" />
+            <StaticImage
+              layout="constrained"
+              width={336}
+              className="built-for-you__image"
+              alt="Man cleaning eyeglasses"
+              src="../../images/index/cleaning-eyeglasses.jpg"
+            />
           </div>
           <h4 className="built-for-you__card-heading">Engineering Leaders</h4>
           <p className="built-for-you__text">Seeking business-minded product development teams</p>
         </div>
         <div className="built-for-you__card">
           <div className="built-for-you__image-container">
-            <img className="built-for-you__image" src="https://placehold.co/336x160" />
+            <StaticImage
+              layout="constrained"
+              width={336}
```

Comment:
one thing to consider is to extract the `cardImageMaxWidth` constant here so the number is not duplicated across the three components.